### PR TITLE
Fill in and fix some authorized links

### PR DESCRIPTION
**Pull request by cavis**
Missing some links:

- [x] Adds `prx:authorization` link to the API root.
- [x] Fix the authorization podcast/episode self urls to stay under `/api/v1/authorization`
- [x] Keep the authorization podcast `prx:episodes` link under `/api/v1/authorization`
- [x] The `/api/v1/authorization/podcasts/1234/episodes` routes were broken

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
       resource :authorization, only: [:show] do
         resources :podcasts, except: [:new, :edit], module: :auth do
-          resources :episodes, except: [:new, :edit], module: :auth
```
Comment (chrisrhoden) [inline config/routes.rb:16]: I confess I don't know enough about rails routes.rb to know why this change with no apparent controller change - is it just redundant because of the parent route?

Comment (cavis) [reply config/routes.rb:16]: It was doubling the `auth` module ... so looking for the `Api::Auth::Auth::EpisodesController`.  (This route has never worked).

Comment (chrisrhoden) [reply config/routes.rb:16]: Got it!

Comment (chrisrhoden) [reply config/routes.rb:16]: Got it!
